src/codeu/chat/client/commandline/Chat.java

```diff
@@ -92,6 +93,18 @@ private Panel createRootPanel(final Context context) {
 
     final Panel panel = new Panel();
 
+    panel.register("info", new Panel.Command() {
```

Comment:
Makes sense! I moved it under the panel.register("help" ... call and put info above the other calls in the System.out.println.